Gruntfile.js

```diff
@@ -131,12 +155,14 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-shell');
   grunt.loadNpmTasks('grunt-jstestdriver-phantomjs');
+  grunt.loadNpmTasks('grunt-closurecompiler');
   grunt.loadTasks('grunt-chrome-build');
 
   // set default tasks to run when grunt is called without parameters
   grunt.registerTask('default', ['csslint', 'htmlhint', 'jscs', 'jshint',
-                     'shell:runPythonTests', 'jstdPhantom']);
+                     'shell:runPythonTests', 'jstdPhantom',
+                     'closurecompiler:debug']);
   grunt.registerTask('build', ['grunt-chrome-build']);
```

Comment:
This step should probably be added to the build step as well. I'm happy to do it after this is in if you don't end up making any other changes.